src/MainPage/MainPage.js

```diff
@@ -133,7 +139,7 @@ function MainPage() {
     pkmnCry.play();
     let newHappiness = Math.min(pokemonData.data.attributes.happiness + 5, 100);
     fetch(
-      "https://obscure-caverns-08355-6f81aa04bbe3.herokuapp.com/api/v1/trainers/1/pokemons/2",
+      "https://obscure-caverns-08355-6f81aa04bbe3.herokuapp.com/api/v1/trainers/1/pokemons/${pokemonData.data.id}",
```

Comment:
Nice catch on this bug, it was strange seeing the object render to one pokemon every time, thankfully it was an easy fix.